lib/linguist/language.rb

```diff
@@ -27,13 +27,10 @@ class Language
     @name_index         = {}
     @alias_index        = {}
     @language_id_index  = {}
-
```

Comment:
And this one. Please can we have it back.

Reply:
done